src/modules/core/components/ActionsList/ActionsListItem.tsx

```diff
@@ -94,6 +104,12 @@ const ActionsListItem = ({
     [handleOnClick, id, transactionHash],
   );
 
+  let domainName;
+  if (metadataJSON && actionType === ColonyActions.EditDomain) {
+    const domainObject = parseDomainMetadata(metadataJSON);
+    domainName = domainObject.domainName;
+  }
+
```

Comment:
I think this is the only thing I could find "wrong" with this PR, the domain name in the actions list shows the "new" name, not the old one that was set when it was created.

Here's how to replicate:
1. create domain -> "domain second"
2. edit domain, from "domain second" -> "domain second plus"

The first entry in the list (the create domain one) should show "domain second", but currently it shows "domain second plus"

However when you click on the action, and go to action page it shows the correct values.

Here's a demo of the bug in action:

![bug-domain-rename-action-list-current-domain](https://user-images.githubusercontent.com/1193222/107114571-819ada80-686f-11eb-9fe9-9ab64348b985.gif)